app/e2e/packetPage.spec.ts

```diff
@@ -0,0 +1,118 @@
+      }
+    });
+
+    test("can see Dependencies section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      const dependenciesDiv = await getPacketPageAccordionSection(content, "Dependencies");
+      // Section should contain either no depencies text, or at least one list item with link
+      const noneDiv = await dependenciesDiv.locator(".italic");
+      if ((await noneDiv.count()) > 0) {
+        await expect(noneDiv).toHaveText("This packet has no dependencies on other packets");
+      } else {
+        const depItems = await dependenciesDiv.getByRole("listitem");
+        await expect(await depItems.count()).toBeGreaterThan(0);
+        for (const li of await depItems.all()) {
+          const link = await li.getByRole("link");
+          await expect(link).toBeEnabled();
+          const linkText = await link.innerHTML();
+          expect((await link.getAttribute("href")).split("/").at(-1)).toBe(linkText);
+        }
+      }
+    });
+
+    test("can see Reports section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      const reportsDiv = await getPacketPageAccordionSection(content, "Reports");
+      // Reports should contain either "None" or at least one iframe
+      const noneDiv = await reportsDiv.locator(".italic");
+      if ((await noneDiv.count()) > 0) {
+        await expect(noneDiv).toHaveText("None");
+      } else {
+        const reportFrames = await reportsDiv.locator("iframe");
+        await expect(await reportFrames).toHaveCount(1);
+      }
+    });
+  });
+
+  test.describe("Metadata", () => {
+    test("can see Timings section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      await selectPacketPageTab(content, "Metadata");
+      const timingsDiv = await getPacketPageAccordionSection(content, "Timings");
+      const timingItems = await timingsDiv.getByRole("listitem");
+      await expect(await timingItems).toHaveCount(2);
+      await expect(await timingItems.first()).toHaveText(/Started/);
+      await expect(await timingItems.last()).toHaveText(/Elapsed\d+/);
+    });
+
+    test("can see Git section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      await selectPacketPageTab(content, "Metadata");
+      const gitDiv = await getPacketPageAccordionSection(content, "Git");
+      // get top level list items
+      const listItems = await gitDiv.locator("ul.space-y-1 > li").all();
+      await expect(await listItems.length).toBe(3);
+      await expect(listItems[0]).toHaveText(/^Branch/);
+      await expect(listItems[1]).toHaveText(/^Commit[\da-f]{40}$/);
+      await expect(await listItems[2].locator("span")).toHaveText("Remotes");
+      const remoteItems = await listItems[2].getByRole("listitem");
+      await expect(await remoteItems.count()).toBeGreaterThan(0);
+    });
+  });
```

Comment:
NB this metadata is optional, so the section may validly not exist: https://github.com/mrc-ide/orderly2/blob/main/inst/schema/outpack/metadata.json